mock/imposter-config.yaml

```diff
@@ -0,0 +1,24 @@
+plugin: openapi
+specFile: https://mongodb-mms-prod-build-server.s3.amazonaws.com/openapi/acc6ed9d3dd14f4b74d68486bf49c906b3cf7318-v1.json
+
+validation:
+  request: true
+  levels:
+    validation.request.body.missing: ERROR
+
+resources:
+  - path: "/api/atlas/v1.0/groups/64f59c4f8db4346b7322b6df/databaseUsers/admin/test"
+    method: get
+    response:
+      statusCode: 200
+      file: databaseUsers.json
+  - path: "/api/atlas/v1.0/groups/64f59c4f8db4346b7322b6df/databaseUsers"
+    method: post
+    response:
+      statusCode: 200
+      file: databaseUsers.json
+  - path: "/api/atlas/v1.0/groups/64f59c4f8db4346b7322b6df/databaseUsers/admin/test"
+    method: patch
+    response:
+      statusCode: 200
+      file: databaseUsers.json
```

Comment:
This section is used to customize the response from the mock server if the autogenerated response is not enough. In our case we want to make sure to return the values used in the acceptance test to make sure that Terraform won't complain about different values between the configuration file and the state.